src/IIIFPresentation/API/Features/Storage/Requests/CreateCollection.cs

```diff
@@ -43,7 +44,6 @@ public async Task<ModifyEntityResult<FlatCollection>> Handle(CreateCollection re
             CustomerId = request.CustomerId,
             IsPublic = request.Collection.Behavior.IsPublic(),
             IsStorageCollection = request.Collection.Behavior.IsStorageCollection(),
-            ItemsOrder = request.Collection.ItemsOrder,
```

Comment:
not sure this is correct -m ight be worth adding this back in?

Reply:
who's best to answer that?

Reply:
talked with Tom, decision is to add